packages/client-sdk-nodejs/src/internal/vector-index-control-client.ts

```diff
@@ -85,6 +89,23 @@ export class VectorIndexControlClient implements IVectorIndexControlClient {
     const request = new grpcControl._CreateIndexRequest();
     request.index_name = indexName;
     request.num_dimensions = numDimensions;
+
+    switch (similarityMetric) {
+      case VectorSimilarityMetric.INNER_PRODUCT:
+        request.inner_product =
+          new grpcControl._CreateIndexRequest._InnerProduct();
+        break;
+      case VectorSimilarityMetric.EUCLIDEAN_SIMILARITY:
+        request.euclidean_similarity =
+          new grpcControl._CreateIndexRequest._EuclideanSimilarity();
+        break;
+      case VectorSimilarityMetric.COSINE_SIMILARITY:
+      default:
+        request.cosine_similarity =
+          new grpcControl._CreateIndexRequest._CosineSimilarity();
+        break;
```

Comment:
If undefined we should default to cosine. We should error check for something out of bounds though.